README.md

```diff
@@ -45,6 +45,103 @@ Required installed programs: Bash and jq for bash, Python3 and the PyVCF, filelo
 
 More CLI examples can be found in the README.md file included in the CLI download or on the [readthedocs page](https://polyriskscore.rtfd.io) for the tool.
 
+## Output Results
+
+There are two choices for the tsv output results - condensed (default) or full. Additonally, you can choose to output results in JSON format, which contains all the information found in the 'full' format. Explanations of the columns found in the output are given below.
+
+- **Study ID** -- The study identifier assigned by the GWAS Catalog (or the user if they uploaded their own GWAS summary statistics)
+- **Reported Trait** -- Trait based on the phenotype being studied, as described by the authors
+- **Trait** -- Trait assigned by the GWAS Catalog, standardized from the Experimental Factor Ontology
+- **Citation** -- The citation of the study
+- **P-Value Annotation** -- Additional information about the p-values
+- **Beta Annotation** -- Additional information about the beta values
+- **Score Type** -- This indicates if the study used odds ratios or beta values
+- **Units (if applicable)** -- This column will contain the beta units if the Score Type is beta. 
+- **SNP Overlap** -- Details the number of SNPs that are in the sample vcf/txt file which are in the study and not excluded from the calculation (see below)
```

Comment:
"...number of SNPs that are in the <input> vcf/txt file which are <also> in the study and not..."